### DISCUSSION TITLE
Переклад слова preset

### DISCUSSION THREAD

**Discussion by denysdesign** [Joomla-Ukraine/uk-UA#19] [Термінологія]
Ваші пропозиції щодо слова preset.

Термін часто вживається в Joomla 4. По суті це можна перекласти як _Попередньо встановлені налаштування_

**Контекст**
https://github.com/Joomla-Ukraine/uk-UA/search?q=%D0%BF%D1%80%D0%B5%D1%81%D0%B5%D1%82

**Comment by Kostelano** [comment]
Напомни, в каком контексте употребляется?

**Reply by denysdesign** [reply]
Додав приклад по локалізації

**Comment by Kostelano** [comment]
На русский язык местами перевели как "Предустановки".

По сути это "набір параметрів". Как вариант.

**Reply by denysdesign** [reply]
"Набір параметрів" подобається, і зрозуміло і стисло
